### PR TITLE
fix: localisation with commas and decimals

### PR DESCRIPTION
**Pull request by Yahhi**
Some fixes for issue #1526

**Changes**: A lot of places where dot separator was used instead of comma separator in local language are fixed. But I can't find all the places yet. Additionally, I removed some occurrences of valueOf, when it is used to parse values from user input
 because it can cause a crash for input with a comma.

**Screenshot/s for the changes**: One example screen:
image

**Checklist**: [Please tick following check boxes with `[x]` if the respective task is completed]
- [x] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [ ] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [ ] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [x] My code does not contain any extra lines or extra spaces
- [ ] I have requested reviews from other members

**APK for testing**: 
[fix_localization_dot_comma_issues.zip](https://github.com/fossasia/pslab-android/files/2751172/fix_localization_dot_comma_issues.zip)

### CODE REVIEW

**Comment by cweitat** [comment]
@Yahhi Hi, please fix Codacy

File: app/src/main/java/io/pslab/activity/WaveGeneratorActivity.java [hunk 1]
```diff
@@ -183,6 +183,11 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_wave_generator_main);
         ButterKnife.bind(this);
+        waveFreqValue.setText(getString(R.string.freq_value, 5.255));
+        waveMonPropValueSelect.setText(getString(R.string.freq_value, 5.255));
+        pwmFreqValue.setText(getString(R.string.freq_value, 5.255));
+        pwmMonPropSelectValue.setText(getString(R.string.freq_value, 5.255));
+        pwmPhaseValue.setText(getString(R.string.freq_value, 5.255));
```
Comment (CloudyPadmal) [inline app/src/main/java/io/pslab/activity/WaveGeneratorActivity.java]: Hi,

Is this for testing purposes? If so, please remove them.

Thanks

**Comment by CloudyPadmal** [comment]
Hello @Yahhi , can you update the branch with development branch?

**Comment by cweitat** [comment]
@Yahhi if there's no replies to this we will close this issue to let others to take up

**Comment by ho-dor** [comment]
@cweitat can i take this up ?

**Pull request closed without merging**